libs/server/Storage/Session/ObjectStore/SetOps.cs

```diff
@@ -552,22 +555,23 @@ public GarnetStatus SetDiffStore(byte[] key, ArgSlice[] keys, out int count)
             return GarnetStatus.OK;
         }
 
-        private (HashSet<byte[]>, GarnetStatus) _setDiff<TObjectContext>(ArgSlice[] keys, ref TObjectContext objectContext)
+        private HashSet<byte[]> SetDiff<TObjectContext>(ArgSlice[] keys, ref TObjectContext objectContext)
             where TObjectContext : ITsavoriteContext<byte[], IGarnetObject, SpanByte, GarnetObjectStoreOutput, long>
         {
-            var result = new HashSet<byte[]>();
+            if (keys.Length == 0)
+            {
+                return [];
```

Comment:
Why not return result here (i.e. default?)

Reply:
I wanted to reduce instantiation, but later on, I realized that there were some unavoidable.